### ISSUE TITLE
did small batch size makes poor result?

### ISSUE THREAD

**Issue by wildbrother** [zylo117/Yet-Another-EfficientDet-Pytorch#675]
I have 4 X RTX Titan.  
I always trained with D3 model because with D4 or D5 models It makes poor result. (extremely)
When I trained with D3 models, I can use 16~20 batch size (fine tune) . 
but in D5 model, I can use only 4 batch size. 

I use 1e-5 learning rate when I fine-tuning your models.
I know training with small learning rate is better when the batch size is very small. 
but 5e-5 or 1e-6 are so small. I think  it's nonsense.

result with my custom data (Open Images 40 classes)

D3 -> MAP(AP50) : 60
D5 -> MAP(AP50) : 1

did small batch size like 4 (one batch for one gpu) makes poor result??

**Comment by wildbrother** [comment]
okay. I found this is the problem of batch norm.  I think sync batch could not solve this problem, if total batch is under 16. 

now I am recifying your code with gradient accumulate & group normalization.  
and I suggest you to change your code like this.

**Comment by zylo117** [comment]
training with small batchsize will cause poor result in effdet. sync batch helps, but lr of 1e-5 is way too small
